File: Java/Arrays/maxConsecutiveOnesBinaryArray.java
import java.util.*;

class maxConsecutiveOnesBinaryArray{

    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of the array : ");
        int n = sc.nextInt();
        ArrayList<Integer> arr = new ArrayList<>();
        System.out.println("Enter the elements of the array : ");
        for(int i=0;i<n;i++){
            int k = sc.nextInt();
            arr.add(k);
        }
        System.out.println("Max consecutive 1's in the given binary array are : " + maxConsecutiveOnes(arr));
    }

    public static int maxConsecutiveOnes(ArrayList<Integer> arr){
        int n = arr.size(), res = 0,  count = 0;
        for(int i=0;i<n;i++){
            if(arr.get(i) == 0){
                count = 0;
            }
            else{
                count = count + 1;
                res = Math.max(res, count);
            }
        }
        return res;
    }
};

File: Python/Arrays/maxConsecutiveOnesBinaryArray.py
def maxConsecutiveOnes(arr):
    n = len(arr)
    res = 0
    count = 0
    for i in range(0, n):
        if(arr[i] == 0):
            count = 0
        else:
            count = count + 1
            res = max(res, count)
    return res

if __name__ == "__main__":
    n = int(input("Enter the size of the array : "))
    arr = list()
    print("Enter the elements of the array : ")
    for i in range(0, n):
        k = int(input())
        arr.append(k)
    print("Max consecutive 1's in the given binary array are : ", maxConsecutiveOnes(arr))